prov/util/src/util_av.c

```diff
@@ -661,6 +661,10 @@ static int ip_av_insert_addr(struct util_av *av, const void *addr,
 
 	if (fi_addr)
 		*fi_addr = !ret ? index : FI_ADDR_NOTAVAIL;
+
+	ofi_straddr_log(av->prov, FI_LOG_DEBUG, FI_LOG_AV, "av_insert addr", addr);
+	FI_DBG(av->prov, FI_LOG_AV, "av_insert fi_addr: %" PRIu64 "\n", *fi_addr);
```

Comment:
FI_DBG compiles to a no-op in a non-debug build. FI_LOG_DEBUG does not.  I like the ofi_straddr_log function, but it's adding overhead in all cases.  For example, it always calls ofi_translate_addr_format(), even if the log message is ignored.  That means it may not be suitable for fast path debugging.

We may need to customize the call more.